content/components/action-menu.mdx

```diff
@@ -466,6 +466,22 @@ See [focus management](/guides/accessibility/focus-management) for more informat
   </Box>
 </Box>
 
+### Inactive menu items vs disabled menu items
+
+Since inactive items can still be focused and activated (for example, to show a modal with more information about an outage), they **should not** be conveyed as disabled using `aria-disabled`.
+
+This behavior is consistent with the behavior of inactive buttons. See the [button component accessibility guidelines](/components/button#inactive-buttons-and-aria-disabled) and [inactive button guidelines](/components/button#inactive) for more details.
+
+#### Inactive menu items:
+
+- Can be activated
+- Need to be able to get focus so screen reader users can "read" the message about it being inactive
+
+#### Disabled menu items:
+
+- Do not get focus
```

Comment:
I'm wondering if this is true in all cases. If a menu item receives `aria-disabled`, wouldn't we want to keep it focusable? Example: 

```html
<ul role="menu">
  <li role="menuitem" tabindex="0" aria-disabled="true">Item 1</li>
  <li role="menuitem" tabindex="-1">Item 2</li>
</ul>
```

Would the markup differ than the example?

Reply:
We might, but currently we only pass `aria-disabled` for disabled items. I don't believe we allow consumers to manually pass `aria-disabled`.